### PR TITLE
Sidebar ver1.0

### PR DESCRIPTION
**Pull request by Erykahh1982**
Sziasztok,

We have decided to ask for  a pull request of my work till now on the sidebar brach, however it is still not works as per expected, BUT the FIXES made on the header and resources components are within this branch and the other team members would need to have these also implemented. Can you please check the code and give feedback on it.

If it will be approved, I'll generate a new brach sidebar2 to continue my progress there.
Thank you
Erika

### CODE REVIEW

File: Tribes/src/app/resources/resources.component.html [hunk 1]
```diff
@@ -3,13 +3,13 @@
   <div class="row justify-content-center">
     <ul>
       <li *ngFor="let resource of resources">
-        <div *ngIf="resource.type==='food'">
```
Comment (balazsbarni) [inline Tribes/src/app/resources/resources.component.html]: szerintem a mostani verzioban a `type` a jo

File: Tribes/src/app/statusboard/statusboard.component.html [hunk 2]
```diff
@@ -1,4 +1,44 @@
 <h1>STATUSBOARD</h1>
 <div>
     <app-resources></app-resources>
+</div>
+
+<div class="container-fluid">
+    <div>
+        <ul>
+            <li *ngFor="let building of listOfBuildings">
+                <div *ngIf="building.buldingTypeENUM ==='MINE'">
```
Comment (balazsbarni) [inline Tribes/src/app/statusboard/statusboard.component.html]: ugyanugy itt is, szerintem az Enumos verzio mar nem OK

File: Tribes/src/app/resources/resources.component.html (hunk 1 above)
Comment (Erykahh1982) [reply Tribes/src/app/resources/resources.component.html]: done

File: Tribes/src/app/statusboard/statusboard.component.html (hunk 2 above)
Comment (Erykahh1982) [reply Tribes/src/app/statusboard/statusboard.component.html]: done

**Pull request merged**